src/services/item/item.schemas.packed.ts

```diff
@@ -31,7 +31,15 @@ export const packedItemSchemaRef = registerSchemaAsRef(
       permission: customType.Nullable(customType.EnumString(Object.values(PermissionLevel))),
       hidden: Type.Optional(itemVisibilitySchemaRef),
       public: Type.Optional(itemVisibilitySchemaRef),
-      thumbnails: Type.Optional(Type.Object({}, { additionalProperties: true })),
+      thumbnails: Type.Optional(
+        customType.StrictObject(
+          {
+            small: Type.String(),
```

Comment:
add format `uri`?

Reply:
what about an item that does not have thumbnails? it doesn't have the `thumbnails` prop?